src/stream.ts

```diff
@@ -131,6 +102,30 @@ export class WebRTCStream implements Stream {
       let err = (evt as RTCErrorEvent).error;
       this.abort(err);
     };
+
+    const self = this;
+    // reader pipe
+    this.channel.onmessage = async ({data}) => {
```

Comment:
are we planning to expand the message object then with more properties?

Reply:
no, that seems unlikely to happen.